pkg/custom-provider/provider.go

```diff
@@ -147,16 +148,18 @@ func (p *prometheusProvider) buildQuery(ctx context.Context, info provider.Custo
 	// TODO: use an actual context
 	queryResults, err := p.promClient.Query(ctx, pmodel.Now(), query)
 	if err != nil {
-		klog.Errorf("unable to fetch metrics from prometheus: %v", err)
+		klog.Errorf("unable to fetch metrics from prometheus: %s", err.Error())
```

Comment:
Can you insert `custom` in between `fetch` and `metrics`? An improvement to more easily disambiguate where these are coming from